Makefile

```diff
@@ -7,6 +7,8 @@ PEM := $(PEM_DIR)/cert.pem
 NAMESERVER := 8.8.8.8
 RESOLV_CONF_DIR := endpoint/endpointComp
 OUTPUT_BASE_DIR := output_base
+TARGET := localhost
```

Comment:
We usually refer "TARGET" as some devices, build goals, places. But `localhost` is intended to be a network loop.

Reply:
Change it to `HOST` or `PRESET_HOST`.

Reply:
The legato uses `localhost` to represent the development host. The `localhost` here is different with the network loop.

Reply:
Clarify with informative comments.